### DISCUSSION TITLE
Information Sources

### DISCUSSION THREAD

**Discussion by mizaki** [comictagger/comictagger#549] [General]
If you would like an information source (with API) added to CT, request it here.

Please provide links to the site and API docs.

**Comment by Roccia7** [comment]
Hello, is it possible to add this sources?
https://comixtime.it/
https://www.comicsbox.it/

**Reply by mizaki** [reply]
Does Comixtime have a public API (and some docs)? I had to use translate and so am not sure on much but I did notice `Our database is protected by copyright and any operation of extraction or reuse of all or part of it is prohibited.` on the DEX site.

Similar for comicsbox, does it have a public API (and some docs)?

**Reply by Roccia7** [reply]
I do not know, sorry😅

**Reply by mizaki** [reply]
Without an API (and permission), I'm afraid it's unlikely.

**Reply by Roccia7** [reply]
Oh I didn't know that, thanks anyway

**Comment by mslmn** [comment]
Would be great to have Amazon as a source if possible.

**Reply by mizaki** [reply]
Amazon doesn't have a public API (afaik). 

Out of curiosity, what is the reason because won't all the same info be in ComicVine?

**Reply by mslmn** [reply]
Sometimes ComicVine takes a while to be updated with new releases, and other more obscure series aren’t on CV at all. I’ve also come across series where some earlier volumes aren’t on ComicVine but newer ones are. With Amazon, new releases have their info available even before their release date.

**Reply by lordwelch** [reply]
This could probably be used https://pypi.org/project/python-amazon-product-api/ though not sure how effective it would be

**Reply by mizaki** [reply]
Last updated in 2014 and the homepage is dead. There is https://webservices.amazon.com/paapi5/documentation/ which is what it is using (although maybe not v5). Either way, an Amazon Associates account is needed which prohibits its use.

Maybe checking what book and music cover downloaders do with Amazon might give a clue (I imagine they scrape).

**Reply by saskir12** [reply]
To revive it from the Graveyard. While CV is nice it certainly lacks timely additions (which quite often can be attested to the mods not greenlighting an edit) and some series are missing on it. Which comes to mind. The New release of Baki the Grappler and anotehr one would be Beat & Motion. Both series are a long time out. Also some times edits are missing even basic information as who the author is. Maybe we can look how Maforget has done so for ComicRack with his Amazon plugin.

**Comment by migube** [comment]
could you add a scraper for Bedetheque like this one?
tx
https://github.com/maforget/Bedetheque-Scrapper-2

**Reply by lordwelch** [reply]
See #625

**Comment by killo3967** [comment]
Hello. First of all, thank you for your work.
I'd like to ask you to consider preparing your plugin system to allow scraping in the future, or at least to communicate with third-party scripts that perform such scraping and return the searched data. As you can see, there are a ton of websites that don't have an API, but they do have a ton of cataloged comics, especially in languages ​​other than English/American.

**Reply by lordwelch** [reply]
Anyone that wants to can make any scraper they want, I'm only going to advertise it if it uses an official API or han permission from the website to scrape it

**Comment by andreasorbelli** [comment]
Hello! Really nice work! 
What about this as a source? [Inducks](https://coa.inducks.org/index.php)

**Reply by lordwelch** [reply]
It's doable, they provide daily exports that can be loaded into a database, and one of the previous maintainers of CT even has some code to work with it https://gist.github.com/davide-romanini
I make no promises as to if or when I get around to it but it's one I'll look at
